extensions/mssql/tsconfig.json

```diff
@@ -7,6 +7,7 @@
 		"strictNullChecks": false
 	},
 	"include": [
-		"src/**/*"
+		"src/**/*",
+		"../../src/vscode-dts/vscode.proposed.textDocumentNotebook.d.ts"
```

Comment:
Normally we add typings files to the refs.d.ts

Reply:
Where does that refs file get used? We only need this typings file in MSSQL

Reply:
Each extension can (and usually does) have its own - that's how they get the vscode/azdata typings pulled in (among any others they use)

https://github.com/microsoft/azuredatastudio/blob/main/extensions/mssql/src/typings/refs.d.ts